Desktop: Resource links broken

### Operating system

Windows

### Joplin version

3.0.0

### Desktop version info

Joplin 3.0.0 (dev, linux)

Client ID: 44da07c87ebf4662aa0e0f97f77f7f38
Sync Version: 3
Profile Version: 46
Keychain Supported: No

Revision: e59211dee

ABC Sheet Music Plugin: 1.0.4
Backup: 1.4.0
CodeMirror 6 snippets: 0.0.5
Freehand Drawing: 2.7.1

### Current behaviour

1. Add a resource link to a note (e.g. with "attach file")
    - I've tested with PDF and Python files.
2. Try to open the link

Currently, this fails with a "missing property: `deleted_time`" error.

### Expected behaviour

Joplin should be able to open resource links.

### Logs

````
08:29:25: Updating items_normalized from {updated_time: 1709687458116, id: 'bb87f4b4a25a499eaa8c8219c5d03186'}
main-html.js:54 08:29:25: models/Setting: Saving settings...
main-html.js:54 08:29:25: models/Setting: Settings have been saved.
main-html.js:54 08:29:25: SearchEngine: Updated FTS table in 174ms. Inserted: 2. Deleted: 0
main-html.js:54 Got ipc-message: joplin://36403c758d9c40c08e8ba550b3b35541 {resourceId: '36403c758d9c40c08e8ba550b3b35541'}
main-html.js:54 08:29:26: CommandService::execute: openItem joplin://36403c758d9c40c08e8ba550b3b35541
main-html.js:54 Error: Missing property "deleted_time": {"id":"36403c758d9c40c08e8ba550b3b35541","title":"output.pdf","mime":"application/pdf","filename":"","created_time":1709828964028,"updated_time":1709828964287,"user_created_time":1709828964028,"user_updated_time":1709828964287,"file_extension":"pdf","encryption_cipher_text":"","encryption_applied":0,"encryption_blob_encrypted":0,"size":6226,"is_shared":0,"share_id":"","master_key_id":"","user_data":"","blob_updated_time":1709828964028,"ocr_text":"this is a test\ntesting\ntest","ocr_details":"","ocr_status":2,"ocr_error":"","type_":4}
    at checkObjectHasProperties (/home/builder/Documents/joplin/packages/utils/dist/object.js:17:19)
    at itemIsReadOnlySync (/home/builder/Documents/joplin/packages/lib/models/utils/readOnly.js:86:43)
    at /home/builder/Documents/joplin/packages/app-desktop/gui/NoteEditor/utils/contextMenu.js:67:55
    at Generator.next (<anonymous>)
    at fulfilled (/home/builder/Documents/joplin/packages/app-desktop/gui/NoteEditor/utils/contextMenu.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
````

--- a/packages/app-desktop/bridge.ts
+++ b/packages/app-desktop/bridge.ts
@@ -288,9 +288,7 @@ export class Bridge {
 	}
 
 	/* returns the index of the clicked button */
-	public showMessageBox(message: string, options: MessageDialogOptions = null) {
-		if (options === null) options = { message: '' };
-
+	public showMessageBox(message: string, options: MessageDialogOptions = {}) {
 		const result = this.showMessageBox_(this.window(), { type: 'question',
 			message: message,
 			buttons: [_('OK'), _('Cancel')], ...options });

